docker/Dockerfile

```diff
@@ -1,80 +0,0 @@
-FROM ubuntu:20.04
```

Comment:
Do we still want to leave this file to support allowing customers of the open source to have a build environment that works for them?